web_ui/dex_templates/static/fonts/intel-one-mono/intel-one-mono.css

```diff
@@ -0,0 +1,55 @@
+@font-face {
+    font-family: 'IntelOne Mono';
+    src: url('./IntelOneMono-Light.woff') format('woff');
+    font-weight: 300; /* Light */
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'IntelOne Mono';
+    src: url('./IntelOneMono-LightItalic.woff') format('woff');
+    font-weight: 300; /* Light */
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'IntelOne Mono';
+    src: url('./IntelOneMono-Regular.woff') format('woff');
+    font-weight: 400; /* Regular */
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'IntelOne Mono';
+    src: url('./IntelOneMono-Italic.woff') format('woff');
+    font-weight: 400; /* Regular */
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'IntelOne Mono';
+    src: url('./IntelOneMono-Medium.woff') format('woff');
+    font-weight: 500; /* Medium */
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'IntelOne Mono';
+    src: url('./IntelOneMono-MediumItalic.woff') format('woff');
+    font-weight: 500; /* Medium */
+    font-style: italic;
```

Comment:
let's go through all of these 1 by 1 and make sure we only add to the project the ones we use. This italic for instance is never used so we dont need to add it.